server/ipfs.js

```diff
@@ -0,0 +1,57 @@
+import ipfsAPI from 'ipfs-api';
+
+const hosts = (process.env.IPFS_HOST || 'localhost').split(',');
+const master = (process.env.IPFS_MASTER || 'localhost').split(',');
+
+const addrs = [];
+const clients = [];
+
+const initClient = async (host) => {
+  const options = {
+    host,
+    port: process.env.IPFS_PORT || '5001',
```

Comment:
We need to change the way that PORT config too. I don't think assuming same IPFS_PORT across all node makes much sense.